app/views/practical_supports/_edit.html.erb

```diff
@@ -1,65 +1,47 @@
+        local: false do |f| %>
+      <%= f.select :support_type, options_for_select(practical_support_options, support.support_type) %>
+      <%= f.select :source,
+                    options_for_select(practical_support_source_options(support.source),
+                                      support.source) %>
+      <%= f.number_field :amount,
+                          value: number_to_currency(f.object.amount.to_f, unit: '', delimiter: ''),
+                          autocomplete: 'off',
+                          prepend: '$',
+                          step: '0.01' %>
+      <%= f.date_field :support_date, autocomplete: 'off' %>
 
       <% if Config.display_practical_support_attachment_url? %>
-        <div class="col-4">
-          <%= f.text_field :attachment_url,
-                            autocomplete: 'off',
-                           hide_label: true,
-                           placeholder: t('activerecord.attributes.practical_support.attachment_url') %>
-        </div>
+        <%= f.text_field :attachment_url,
+                          autocomplete: 'off',
+                          placeholder: t('activerecord.attributes.practical_support.attachment_url') %>
       <% end %>
 
-      <div class="col-3">
-        <%= f.form_group :confirmed do %>
-          <%= f.check_box :confirmed,
-                          id: "practical_support_confirmed_#{practical_support.id}",
-                          label_class: 'tooltip-header-checkbox',
-                          data: { 'tooltip_text': practical_support_confirmed_help_text } %>
-          <%= f.check_box :fulfilled,
-                          id: "practical_support_fulfilled_#{practical_support.id}",
-                          label_class: 'tooltip-header-checkbox',
-                          data: { 'tooltip_text': practical_support_fulfilled_help_text } %>
-        <% end %>
-      </div>
-    </div>
-  <% end %>
-
-  <div class="col-12">
-      <%= button_to t('patient.practical_support.remove'),
-                patient_practical_support_path(patient, practical_support),
-                remote: true,
-                method: :delete,
-                data: { confirm: t('patient.practical_support.delete_confirm') },
-                class: "btn btn-danger float-right practical-support-remove" %>
+      <%= f.form_group :confirmed do %>
+        <%= f.check_box :confirmed,
+                        id: "practical_support_confirmed_#{support.id}",
+                        label_class: 'tooltip-header-checkbox',
+                        data: { 'tooltip_text': practical_support_confirmed_help_text } %>
+        <%= f.check_box :fulfilled,
+                        id: "practical_support_fulfilled_#{support.id}",
+                        label_class: 'tooltip-header-checkbox',
+                        data: { 'tooltip_text': practical_support_fulfilled_help_text } %>
+      <% end %>
+    <% end %>
   </div>
-  <hr/>
 </div>
+
+<div class="row">
+  <div class="col">
```

Comment:
I realize that we don't actually _need_ a `Close` button here, but I would really like one.Right now my muscle memory instinct is to click the button when I'm done, so better it be another way to exit the modal.

Reply:
I think super reasonable